src/ui-test/extensionUITest.ts

```diff
@@ -0,0 +1,94 @@
+            expect(viewControls.map(item => item.getTitle())).to.include(AdaptersConstants.RSP_CONNECTOR_NAME);
+            const serversView = new ActivityBar().getViewControl(AdaptersConstants.RSP_CONNECTOR_NAME);
+            expect(serversView).not.undefined;
+            const serversBar = serversView.openView();
+            expect(serversBar).not.undefined;
+            expect(await (await serversBar).getTitlePart().getTitle()).to.equal(AdaptersConstants.RSP_ATIVITY_BAR_TITLE);
+        });
+
+        it('Action button from Servers activity bar is available', async function() {
+            this.timeout(5000);
+            const serversView = new ActivityBar().getViewControl(AdaptersConstants.RSP_CONNECTOR_NAME);
+            const serversBar = await serversView.openView();
+            const titlePart = serversBar.getTitlePart();
+            const actionsButton = await titlePart.getActions();
+            expect(actionsButton.length).to.equal(1);
+            expect(actionsButton[0].getTitle()).to.include('Create New Server');
+        });
+
+        it('Servers tab is available under Explorer bar', async function() {
+            this.timeout(5000);
+            const explorerView = new ActivityBar().getViewControl('Explorer');
+            expect(explorerView).not.undefined;
+            const bar = await explorerView.openView();
+            const content = bar.getContent();
+            const sections = await content.getSections();
+            expect(await Promise.all(sections.map(asd => asd.getTitle()))).to.include(AdaptersConstants.RSP_SERVERS_LABEL);
+            const section = await content.getSection(AdaptersConstants.RSP_SERVERS_LABEL);
+            expect(section).not.undefined;
+            expect(await section.getTitle()).to.equal(AdaptersConstants.RSP_SERVERS_LABEL);
+            const actionsButton = await section.getActions();
+            expect(actionsButton.length).to.equal(1);
+            expect(actionsButton[0].getLabel()).to.equal(AdaptersConstants.RSP_SERVER_ACTION_BUTTON);
+        });
+
+        after(async function() {
+            this.timeout(4000);
+            if (sideBar && await sideBar.isDisplayed()) {
+                sideBar = await new ActivityBar().getViewControl('Extensions').openView();
+                const actionButton = await sideBar.getTitlePart().getAction('Clear Extensions Input');
+                await actionButton.click();
+                view.closeView();
+            }
+            const quickBox = await new Workbench().openCommandPrompt();
+            if (quickBox && await quickBox.isDisplayed()) {
+                await quickBox.cancel();
+            }
+            if (sideBar && await sideBar.isDisplayed()) {
+                await view.closeView();
+            }
+        });
+    });
+}
+
+async function verifyCommandPalette() {
+    const quick = await new Workbench().openCommandPrompt();
+    await quick.setText(`>${AdaptersConstants.RSP_COMMAND} Download server`);
```

Comment:
this only checks a single command, by the name of the test case I'd think it would check all server commands